tests/test_integration/conftest.py

```diff
@@ -1,6 +1,51 @@
+import os
+
+import pytest
+from dash.testing.composite import DashComposite
 from selenium.webdriver.chrome.options import Options
 
 
+class Helpers:
+    """A class to group helpful things when writing tests.
+
+    Added as a pytest fixture.
+
+    Example
+    -------
+    ```
+    def test_my_thing(helpers)
+        helpers.help_me()
+    ```
+    """
+
+    @staticmethod
+    def debug_screenshot(dash_duo: DashComposite, name: str = "") -> None:
+        """Save a screenshot of the current state of the testing server.
+
+        Useful for debugging and developing tests.
+        Saves to .test_debug_screenshots or
+        .test_debug_screenshots/<name> if `name` is provided.
+
+        Parameters
+        ----------
+            dash_duo : DashComposite
+                The current dash_duo fixture.
+            name: str, optional
+                A name (to help you find the screenshots.)
+        """
+        screenshots_dirname = ".test_helper_screenshots/"
+        if name:
+            screenshots_dirname += f"{name}/"
+        os.makedirs(screenshots_dirname, exist_ok=True)  # mkdir -p
```

Comment:
use `pathlib` instead? we would avoid adding the backlashes (potentially platform dependent?)
 
we sort of prefer it over [as you know :P](https://towardsdatascience.com/dont-use-python-os-library-any-more-when-pathlib-can-do-141fefb6bdb5)

more info on pathlib mkdir [here](https://docs.python.org/3/library/pathlib.html?highlight=mkdir#pathlib.Path.mkdir)